sources/platform/runtime/machine-image/ami-overview.md

```diff
@@ -40,33 +40,29 @@ You can click on any image below to see what is pre-installed:
 | [v5.3.2](ami-v532/)        | Mar 11, 2017    |
 | [Stable](ami-stable/)      | Feb 19, 2016 (deprecated) |
 
-In most cases, the default Machine image set for your Subscription will be sufficient to run your builds. The main reasons why you might want to consider
-changing to a more recent image are:
+In most cases, the default Machine image set for your Subscription will be sufficient to run your builds. The main reasons why you might want to consider changing to a more recent image are:
 
 -  You need a newer language/service/package version
 -  You need a newer Docker version
 
 <a name="view-machine-image"></a>
-## Viewing Subscription Machine Image
 
-To see what Machine Image is being used for your subscription:
+## Viewing Subscription Machine Images
 
-* From the left sidebar menu, click on your Subscription
+When a subscription is imported in your Shippable account for the very first time, a [Node pool](/platform/management/subscription/node-pools/) is automatically created for you. The default Node pool has a single Ubuntu 16.04 on-demand node, with the latest released Ubuntu 16.04 machine image. A subscription can also have multiple Node pools that you can create from the billing page. Each node pool specifies the machine image in its `Runtime Version` setting.
```

Comment:
- users cant create node pools from subscriptions billing page. the flow is buy sku from billing -> go to node pools page -> add node pools -> add/allocate nodes
- the machine images are present in the page where node pools are being added